datacapturing/src/main/java/de/cyface/datacapturing/DataCapturingService.java

```diff
@@ -982,13 +1014,54 @@ public void onBindingDied(final @NonNull ComponentName name) {
         @Override
         public void handleMessage(final @NonNull Message msg) {
```

Comment:
This method should be split into at least two submethods.

Reply:
done